app/models/tag.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: tags
+#
+#  id         :bigint           not null, primary key
+#  name       :string(256)      not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Tag < ApplicationRecord
+  has_many :taggings, dependent: :destroy
+
+  scope :alphabetized, -> { order(:name) }
```

Comment:
...and this should have the `by_type` scope instead of Tagging.

Reply:
Moved `by_type` scope to Tag in 413a6e07af1f789775ff19f504d360a3d566d880.